backend/models/pydantic_models.py

```diff
@@ -1,10 +1,6 @@
-import uuid
-from typing import Optional
-from pydantic import BaseModel
+from sqlmodel import SQLModel
 
-class User(BaseModel):
-    id: uuid.UUID
-    email_address: str
-    first_name: str
-    last_name: str
-    password: str
\ No newline at end of file
+
+class Node(SQLModel, table=False):
```

Comment:
We will have to expand upon what the Node stores as its data at a later time. For now this is good to get basic implementation done.